app/javascript/controllers/informatives_controller.js

```diff
@@ -0,0 +1,93 @@
+import { Controller } from "@hotwired/stimulus"
+import accessibleAutocomplete from "accessible-autocomplete"
+
+export default class extends Controller {
+  static targets = ["informatives-container", "select", "output"]
+
+  addInformative(event) {
+    event.preventDefault()
+    this.setInformativeInput()
+    this.submitForm(event)
+  }
+
+  setInformativeInput() {
+    const chosenInformative = document.getElementById("informatives").value
+    const manualTitleInput = document.getElementById("manual-title-input").value
+    const hiddenTitleField = document.getElementById("hidden-title-field")
+
+    const manualTextInput = document.getElementById("manual-text-input").value
+    const hiddenTextField = document.getElementById("hidden-text-field")
```

Comment:
Would setting these elements as Stimulus targets make this simpler perhaps? I can't really decide, maybe it just means moving the complexity around.